openshift/buildConfig.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: si-bandit-buildconfig
+  annotations:
+    description: >
+      This is Thoth Bandit Security Indicator BuildConfig, this template is meant to be used by Bots, but could also be
+      used by humans...
+    openshift.io/display-name: "Thoth: si-bandit BuildConfig"
+    version: 0.4.2
+    tags: thoth,ai-stacks,si-bandit
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station
+    template.openshift.io/long-description: >
+      This is Thoth Bandit Security indicator BuildConfig, this template is meant to be
+      used by Bots, but could also be used by humans...
+    template.openshift.io/provider-display-name: "Red Hat, Inc."
+    thoth-station.ninja/template-version: 0.4.2
+  labels:
+    template: si-bandit-buildconfig
+    app: thoth
+    component: si-bandit
+
+parameters:
+  - description: Git repository for Thoth's Adviser
+    displayName: Thoth Adviser git repository
+    required: true
+    name: GITHUB_URL
+    value: "https://github.com/thoth-station/si-bandit"
+
+  - description: Git repository for bandit security indicator
+    displayName: Thoth Adviser git reference
+    required: true
+    name: GITHUB_REF
+    value: "master"
+
+  - description: Tag of the output ImageStream the resulting container image should go to
+    displayName: ImageStream Tag
+    required: true
+    name: IMAGE_STREAM_TAG
+    value: "latest"
+
+  - description: Thamos Advise setup tag value
+    displayName: THOTH_ADVISE
+    required: true
+    name: THOTH_ADVISE
+    value: "1"
```

Comment:
Maybe `THOTH_DRY_RUN` so we do not use the recommended stack during the build but rather let bots operate on the repo?

Reply:
BTW this might be still relevant. We want to do recommendations on github repo, not during the build (for thoth components).

Reply:
I added it below :)

Reply:
I added it below :)

Reply:
Oh, nice in that case :) :+1: GitHub was not showing me "outdated" comment. :crying_cat_face: